controller-server/src/test/java/com/yahoo/vespa/hosted/controller/deployment/ApplicationPackageBuilder.java

```diff
@@ -337,23 +337,15 @@ private static byte[] buildMeta(Version compileVersion) {
     }
 
     public ApplicationPackage build() {
-        return build(false);
-    }
-
-    public ApplicationPackage build(boolean useApplicationDir) {
-        String dir = "";
-        if (useApplicationDir) {
-            dir = "application/";
-        }
         ByteArrayOutputStream zip = new ByteArrayOutputStream();
         try (ZipOutputStream out = new ZipOutputStream(zip)) {
             out.setLevel(Deflater.NO_COMPRESSION); // This is for testing purposes so we skip compression for performance
-            writeZipEntry(out, dir + "deployment.xml", deploymentSpec());
-            writeZipEntry(out, dir + "validation-overrides.xml", validationOverrides());
-            writeZipEntry(out, dir + "search-definitions/test.sd", searchDefinition());
-            writeZipEntry(out, dir + "build-meta.json", buildMeta(compileVersion));
+            writeZipEntry(out, "deployment.xml", deploymentSpec());
+            writeZipEntry(out, "validation-overrides.xml", validationOverrides());
+            writeZipEntry(out, "search-definitions/test.sd", searchDefinition());
```

Comment:
`search-definitions`? Shouldn't it be either `schemas` or `searchdefinitions` (if there is some backwards compatibility issue stopping us from using `schemas`). Or maybe it's not used at all in controller, if so it can be removed.

Reply:
No issue changing this I guess, I didn't look for it.